srs_data/generate_baseline_report.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env pythona
```

Comment:
This is only needed if you want to run this script as: `$ generate_baseline_report.py`, i.e. omit the `python ` part.